test/squire.spec.js

```diff
@@ -245,6 +246,97 @@ describe('Squire RTE', function () {
         });
     });
 
+    describe('multi-level lists', function () {
```

Comment:
I'm wondering if our implementation of TAB is screwing with the multi-level list behavior